life_expectancy/strategies.py

```diff
@@ -0,0 +1,210 @@
+"""Module for loading/saving data"""
+from pathlib import Path
+from typing import List
+from enum import Enum
+from abc import ABC, abstractmethod
+import pandas as pd
+from pandas import DataFrame
+
+
+class Region(Enum):
+    """
+    Possible regions in life expectancy data.
+    """
+    AT = 'AT'
+    BE = 'BE'
+    BG = 'BG'
+    CH = 'CH'
+    CY = 'CY'
+    CZ = 'CZ'
+    DK = 'DK'
+    EE = 'EE'
+    EL = 'EL'
+    ES = 'ES'
+    EU27_2020 = 'EU27_2020'
+    FI = 'FI'
+    FR = 'FR'
+    HR = 'HR'
+    HU = 'HU'
+    IS = 'IS'
+    IT = 'IT'
+    LI = 'LI'
+    LT = 'LT'
+    LU = 'LU'
+    LV = 'LV'
+    MT = 'MT'
+    NL = 'NL'
+    NO = 'NO'
+    PL = 'PL'
+    PT = 'PT'
+    RO = 'RO'
+    SE = 'SE'
+    SI = 'SI'
+    SK = 'SK'
+    DE = 'DE'
+    DE_TOT = 'DE_TOT'
+    AL = 'AL'
+    EA18 = 'EA18'
+    EA19 = 'EA19'
+    EFTA = 'EFTA'
+    IE = 'IE'
+    ME = 'ME'
+    MK = 'MK'
+    RS = 'RS'
+    AM = 'AM'
+    AZ = 'AZ'
+    GE = 'GE'
+    TR = 'TR'
+    UA = 'UA'
+    BY = 'BY'
+    EEA30_2007 = 'EEA30_2007'
+    EEA31 = 'EEA31'
+    EU27_2007= 'EU27_2007'
+    EU28 = 'EU28'
+    UK = 'UK'
+    XK = 'XK'
+    FX = 'FX'
+    MD = 'MD'
+    SM = 'SM'
+    RU = 'RU'
+
+    @classmethod
+    def get_countries(cls) -> List[str]:
+        """
+        Filters valid countries.
+        :return: A list of country codes representing valid countries.
+        """
+        not_countries = [
+            'EU27_2020',
+            'DE_TOT',
+            'EA18',
+            'EA19',
+            'EFTA',
+            'EEA30_2007',
+            'EEA31',
+            'EU27_2007',
+            'EU28'
+        ]
+        countries = [country.value for country in cls if country.value not in not_countries]
+        return countries
```

Comment:
Normalize use of country/region in the class
```suggestion
    def get_countries(cls) -> List[str]:
        """
        Filters valid countries.
        :return: A list of region codes representing valid countries.
        """
        not_countries = [
            'EU27_2020',
            'DE_TOT',
            'EA18',
            'EA19',
            'EFTA',
            'EEA30_2007',
            'EEA31',
            'EU27_2007',
            'EU28'
        ]
        countries = [region.value for region in cls if region.value not in not_countries]
        return countries
```